### DISCUSSION TITLE
Is it possible to configure the `hd` parameter for OIDC?

### DISCUSSION THREAD

**Discussion by iainsproat** [datalust/seq-tickets#2231] [Help]
Thank you for making OIDC available for all plans for seq.

With OIDC for Google it is possible to limit login to a particular domain by providing the `hd` parameter: https://developers.google.com/identity/openid-connect/openid-connect#hd-param
 
Is it possible to configure this parameter in seq, and how would I go about configuring it?

Thank you in advance.

**Comment by nblumhardt** [comment]
Hi!

The `hd` parameter doesn't appear to limit login to that domain - only optimize the login UI to accept those domains. Configuring app permissions in the identity provider (or simply avoiding automatic account provisioning) should control access effectively.

There's no way currently to get Seq to pass `hd` in the login page redirect.

Hope this helps!

**Reply by iainsproat** [reply]
Thanks @nblumhardt - I've an additional question on this subject which I've opened a separate discussion for: https://github.com/datalust/seq-tickets/discussions/2233
